src/SmartComponents/BaselinesPage/CreateBaselineModal/CreateBaselineModal.js

```diff
@@ -142,7 +148,6 @@ export class CreateBaselineModal extends Component {
                     await createBaseline(newBaselineObject);
```

Comment:
There may be more to this. 

`createBaseline` is not an async function, it is a redux action, but here it is not really dispatched, it is just called, so redux is never involved in this.

The proper fix should involve removing the `await`s here and properly dispatch the redux `createBaseline` action. 

The fix should also not involve `componentsDidUpdate`.

Reply:
@bastilian It is dispatched here https://github.com/RedHatInsights/drift-frontend/blob/3b54a28b5a267de1901acb9747c4022f93cd4968/src/SmartComponents/BaselinesPage/CreateBaselineModal/CreateBaselineModal.js#L430

Also, with React 18, await doesn't act the same way. The navigate gets called before the `await createBaseline` gets finished so it navigates to `undefined`. Using `componentDidUpdate` to wait for the baselineData to come through is the best way to ensure that the `createBaseline` completed.